.github/workflows/zxcron-extended-test-suite.yaml

```diff
@@ -56,7 +56,7 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
-          fetch-depth: '0'
+          fetch-depth: "0"
```

Comment:
Are these intentional changes?

Reply:
Ah no, my spotless did that somehow...reverted @Neeharika-Sompalli , tyty.